packages/core/src/lib/__tests__/code-snippet.spec.svelte

```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
+import CodeSnippet from '$lib/code-snippet.svelte';
+</script>
+
+<CodeSnippet
+  language="json"
```

Comment:
Something clever I saw in Svelte highlight package was to pass the language file itself, rather than a string, as the `language` prop. No need to deal with it in this PR, but I liked the idea in general

```ts
<script lang="ts">
import CodeSnippet from '$lib/code-snippet.svelte';
import json from 'prism/components/prism-json.min.js';
</script>

<CodeSnippet language={json} />
```

Reply:
Interesting, but right now all of the `prismjs` stuff is typed as `any`, and I don't know if I love having a `export let language: any;` prop vs just accepting a string.